### ISSUE TITLE
Not loading

### ISSUE THREAD

**Issue by NikoNiyazi** [pyscript/pyscript#430]
The screenshot explains everything. I use firefox but last time i used it on firefox no problems occured.
![resim](https://user-images.githubusercontent.com/71526347/169361466-fa077302-00e7-442e-9f3d-4babaa179882.png)

**Comment by marianoweber** [comment]
Duplicate #427

**Comment by NikoNiyazi** [comment]
> Duplicate #427

no, the case was different for them. some of the page was loaded.

**Comment by pww217** [comment]
The type error was the same. Some caching may be at play. 

Is this issue still happening or fixed?

**Comment by NikoNiyazi** [comment]
> The type error was the same. Some caching may be at play.
> 
> Is this issue still happening or fixed?

Ok, it works now! Thank you.
